sub/sd_lavc.c

```diff
@@ -103,17 +103,19 @@ static int init(struct sd *sd)
 
     switch (cid) {
     case AV_CODEC_ID_DVB_TELETEXT: {
-        int64_t format;
+        int64_t format = -1;
         int ret = av_opt_get_int(ctx, "txt_format", AV_OPT_SEARCH_CHILDREN, &format);
-        // format == 0 is bitmap
-        if (!ret && format)
+        // libzvbi_teletextdec: format == 0 is bitmap
+        if (ret || format)
```

Comment:
Why? Now if `txt_format` is not an option, i.e. the default state, it will fail probe?

Reply:
Intended. If we can't be sure it's `libzvbi_teletextdec` (or something compatible) it might even not return bitmaps at all, which would break the subs.

Reply:
The assumption was that it's bitmap unless explicitly stated otherwise, which is also default in ffmpeg.

Reply:
stated where? and why would the option be unavailable then?
I really don't see a realistic case where this would break.

Reply:
> stated where?

In documentation of libzvbi_teletextdec which uses bitmaps as default output. It's fine either way probably, it will just try to do text output if another unknown decoder is used that act differently. Not that it is likely to be ever added in ffmpeg anyway.